modules/special-rules/pages/flak.adoc

```diff
@@ -1,2 +1,62 @@
 = Flak
 
+These units are specialised to shoot down enemy flyers and drop pods.
+Some use very sophisticated targeting devices and fast, accurate weapons systems.
+Others simply fill the air with so many munitions that they can hardly fail to get some hits.
+
+---
+
+{flak} units can attack flyers in one of two different ways, depending on what orders you give to their detachment.
+
+[IMPORTANT]
+.Flak and drop pods
+====
+{flak} units may fire on drop pods as if they were flyers, against the Armour value of the transported unit.
+====
+
+== Flak orders
+For optimum effectiveness, {flak} units and their detachment need to be at a suitable state of readiness.
+We represent this state with special orders called Flak.
+
+. If you wish to place any detachments on Flak orders, you must do this in the Start phase.
++
+[TIP]
+====
+You can place a detachment on both Flak orders and Preparatory Bombardment orders at the same time in the Start phase, if you wish, declaring both at the same time.
```

Comment:
```suggestion
You can place a detachment on both Flak orders and Preparatory Bombardment orders at the same time in the Start phase, if you wish.
You must declare both at the same time.
```
I'm not sure why you'd want to make one long sentence out of this when two shorter ones are easier to digest.
Note also [-ing words](https://epicremastered.com/contributors-guide/a-to-z-of-style.html#_ing_words).
Oh, and I've made two lines here per the best practice [One sentence per line](https://epicremastered.com/contributors-guide/best-practices.html#_one_sentence_per_line).